raiden/tests/unit/test_pfs_integration.py

```diff
@@ -790,7 +790,7 @@ def test_insufficient_payment(query_paths_args, valid_response_json):
 
     # PFS fails to return info
     assert_failed_pfs_request(
-        query_paths_args, [insufficient_response], expected_requests=1, expected_get_iou_requests=2
+        query_paths_args, [insufficient_response], expected_requests=1, expected_get_iou_requests=1
```

Comment:
@palango can you especially look at this change from 2 to 1? I don't quite get why the above change would change the behaviour in the iou request mechanism